hardhat-example/scripts/deployContract.js

```diff
@@ -0,0 +1,13 @@
+const hre = require('hardhat');
+
+module.exports = async () => {
+  const provider = new hre.ethers.providers.JsonRpcProvider('http://localhost:7546');
+  const wallet = new hre.ethers.Wallet(process.env.OPERATOR_PRIVATE_KEY, provider);
+  const Greeter = await hre.ethers.getContractFactory('Greeter', wallet);
+  const greeter = await Greeter.deploy('initial_msg');
+  const contractAddress = (await greeter.deployTransaction.wait()).contractAddress;
+
+  console.log(`Greeter deployed to: ${contractAddress}`);
```

Comment:
Please print both Hedera ContractId and Solidity Address if possible.
Not sure which format it's in but note which in the log